### ISSUE TITLE
Menu | Add ariaLabelledBy and ariaLabel

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#13253]
add ariaLabel and ariaLabelledBy to improve accessibility

**Comment by SoyDiego** [comment]
> add ariaLabel and ariaLabelledBy to improve accessibility

Great you will add this part. In my work we are waiting so much for this feature.
I created the PR https://github.com/primefaces/primeng/pull/13167, I can added ariaLabelledBy if is necessary.

Thanks for your support and the work of all your Team!

**Comment by cetincakiroglu** [comment]
> 

Hi @SoyDiego,

Your PR is not wrong but not correct too, since we're following the accessibility implementation in the PrimeVue, the aria-label and aria-labelledby properties should be added to list items instead of anchor tags because the anchor tags will have aria-hidden. I've reworked the menu component and added them here, it'll be available on the 16.0.3 release. https://github.com/primefaces/primeng/commit/cd1f6e5ca0a5e57618382806912ba23e95d07d7d

We appreciate your effort and support, but if you want to contribute to accessibility please check the PrimeVue as an example.

**Comment by cetincakiroglu** [comment]
fixed in https://github.com/primefaces/primeng/pull/13372
